test/integration/dashboard_test.rb

```diff
@@ -16,7 +16,7 @@ def assert_dashboard_link(text)
     within "li[data-name='Host Configuration Status']" do
       click_link(text)
     end
-    assert_current_path hosts_path, :only_path => true
+    assert_current_path hosts_path, :ignore_query => true
```

Comment:
Replace deprecated only_path with ignore_query

Reply:
no need afaict for `:ignore_query` here, `:only_path => true` is simply redundant on *_path helpers.

Reply:
Fixed: `DEPRECATED: The :only_path option is deprecated in favor of the :ignore_query option`

https://github.com/teamcapybara/capybara/commit/4f766066a386bd38b3e3563367ff1921120f6076#diff-9d70bf9a727148288a52a5bb6a1923a8